activesupport/lib/active_support/cache.rb

```diff
@@ -680,6 +690,13 @@ def save_block_result_to_cache(name, options)
           write(name, result, options)
           result
         end
+
+        def ensure_connection_pool_added!
```

Comment:
can you use this method also in the memcache store?

Reply:
done.